_redirects

```diff
@@ -478,3 +478,11 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
 /guides/getting-started/learning-more/getting-started-dbt-core /docs/develop/getting-started/getting-started-dbt-core 301
 /guides/getting-started/learning-more/refactoring-legacy-sql /docs/develop/getting-started/learning-more/refactoring-legacy-sql 301
 /guides/getting-started/learning-more/using-jinja /docs/develop/getting-started/using-jinja 301
+
+
+/docs/dbt-cloud/cloud-ide/the-dbt-ide  /docs/develop/dbt-cloud-features 301!
+/docs/dbt-cloud/cloud-ide/handling-merge-conflicts  /docs/collaborate/git/resolve-merge-conflicts 301!
+/dbt-cloud/cloud-ide/viewing-docs-in-the-ide  /docs/develop/develop-in-the-cloud 301!
```

Comment:
@matthewshaver : just a heads up since you own Git issue #2082, mirna took care of the `viewing-docs-in-the-ide` redirect